src/lib.rs

```diff
@@ -73,17 +91,22 @@ impl IntoIterator for &Pages {
     type IntoIter = Pages;
 
     fn into_iter(self) -> Pages {
-        self.clone()
+        Pages {
+            offset: 0,
+            length: self.length(),
+            per_page: self.per_page(),
+            html_function: self.html_function,
+        }
     }
 }
 
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq, Default)]
 pub struct Page {
     pub offset: usize,
     pub length: usize,
-    pub begin: usize,// total count of the posts, number of items per page, 0 index, 1 index, next arrow and previous arrow, iterator the paginator(advanced)
-    pub end: usize, // current page number
-    html: String, // end is not needed , offset and begin , selector to indicate active or not -> vector of an hash set
+    pub begin: usize,
+    pub end: usize,
+    html: String,
```

Comment:
`html` is defined as a private field but I didn't see how user can use this field

Reply:
It's fixed